docs/topic_guides/choosing_comparators.ipynb

```diff
@@ -0,0 +1,1249 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Choosing String Comparators\n",
+    "\n",
+    "When building a Splink model, one of the most important aspects is defining the [`Comparisons`](../comparison.md) and [`Comparison Levels`](../comparison_level.md) that the model will train on. Each `Comparison Level` within a `Comparison` should contain a different amount of evidence that two records are a match, which the model can assign a Match Weight to. When considering different amounts of evidence for the model and it is helpful to explore fuzzy matching as a way of distinguishing strings that are similar, but not the same, as one another.\n",
+    "\n",
+    "This guide is intended to show how Splink's string comparators perform in different situations in order to help choosing the most appropriate comparator for a given column as well as the most appropriate threshold (or thresholds)."
```

Comment:
Add something to the end that links to the string comparators guide, like... 

"For descriptions and examples of each string comparators available in Splink, see the dedicated [topic guide](comparators.md)."